packages/core/src/components/DatePicker/DatePicker.tsx

```diff
@@ -24,14 +24,22 @@ export const DatePicker: React.FC<Props> & WithStyle = React.memo(
                 popoverPlacement,
                 minSelectableDate,
                 maxSelectableDate,
+                mobile,
                 ...restProps
             } = props,
             id = props.id || props.label.toLowerCase().replace(/\s/g, '') || 'medly-datepicker', // TODO:- Remove static ID concept to avoid dup ID
             date: Date | null = useMemo(
                 () => (value instanceof Date ? value : typeof value === 'string' ? parseToDate(value, displayFormat) : null),
                 [value, displayFormat]
             );
+        const convertDate = (d: string) => {
```

Comment:
You will not need this if you use `MM/dd/yyyy` as default displayFormat for mobile devices.

Reply:
Regardless of display format input fields with type=date store in value field as "YYYY-MM-DD"
> A DOMString representing a date in YYYY-MM-DD format, or empty

 https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/date